gluon-mesh-vpn-wireguard-vxlan/files/lib/gluon/upgrade/400-mesh-vpn-wireguard

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+rm /etc/config/wireguard
+touch /etc/config/wireguard
```

Comment:
`echo "" > /etc/config/wireguard`

Reply:
What's the difference for this use case?

Reply:
Probably shorter and less commands?

Reply:
Oh, yeah. Missed the `rm`.

Reply:
Having a second look at this, it will also discard the wireguard disabled state, which is determined by the user in config-mode.